app/views/navbar.scala.html

```diff
@@ -11,9 +11,30 @@
 
 @import views.html.bootstrap._
 
-    <!-- Fixed navbar -->
+<!-- Fixed navbar -->
 <nav id="header" class="navbar navbar-default navbar-fixed-top">
     <div id="header-container" class="navbar-container container-fluid">
+        <!-- Banner is only visible if user is on test server-->
+        @if(Play.configuration.getString("environment-type").get == "test") {
+            <link href = "https://fonts.googleapis.com/icon?family=Material+Icons" rel = "stylesheet">
```

Comment:
Can you edit the HTML to remove the space around the `=`? I've just added this to the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide) as well.

Reply:
Done!